### PR TITLE
TCF update: deprecate global scope, OOB

### PR DESCRIPTION
**Pull request by anderagakura**
Deprecation of Global Scope, OOB and euconsent-v2 cookie associated with the consensu.org domain

### CODE REVIEW

**Comment by anderagakura** [comment]
Review on going with IAB Europe team

File: TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md [hunk 1]
```diff
@@ -183,7 +178,6 @@ A TC String that contains positive consent signals must not be created before cl
 
 There are two main contexts in which a TC String can be created:
```
Comment (dmdabbs) [inline TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md:179]: @anderagakura suggest you resolve this lingering reference to "global" such as this one ("two main contexts") and later "service-specific or global," &c.

Comment (anderagakura) [reply TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md]: @dmdabbs new update (https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/pull/301/commits/48a7e378d969fbf903f23b7187bb95e198f983f8)

File: TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md [hunk 2]
```diff
@@ -179,16 +174,9 @@ A Transparency & Consent String may only be created by an IAB Europe TCF registe
...
-
-If the disclosures do not describe a global scope, or explicitly state service-specific processing, then the TC String is used and stored explicitly as a service-specific string. Also, if the CMP discloses transparency and consent in a global context but the user’s browser does not permit third-party cookies, then the CMP’s only recourse is to retain the user’s preference using a local storage mechanism (eg. first-party cookie or [window.localStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage)). Since the transparency and consent obtained from the user is restricted to that site or service, the TC String must then have the service-specific bit [IsServiceSpecific](#tc-string-format) set.
+CMPs must be set up to operate in a **service-specific**. A  TC String in this context is only used by the site(s) or app(s) on which it is running. One is created for every user on a given site/app or group of sites/apps. They may contain [Publisher restrictions](#what-are-publisher-restrictions), a _**[Publisher TC](#publisher-purposes-transparency-and-consent)**_ segment and an _**[AllowedVendors](#allowed-vendors-oob)**_ segment.
```
Comment (a2intl) [inline TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md:179]: remove `be set up to`, and add the word "scope" after `**service-specific**`

File: TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md [hunk 3]
```diff
@@ -360,9 +335,7 @@ The following details provide information on creating, storing, and managing a T
...
-
-It is important to note that with the creation of the version 2 TCF Specifications globally-scoped and service-specific scoped TC Strings have different encoding and decoding requirements.  Some segments are not allowed in a global scope and some are not allowed in a service-specific scope. This document attempts to call out those differing requirements explicitly where applicable.
+In version 1 of the TCF Specifications the consent string was specified to be stored as either a 1st party cookie for service-specific consent or a 3rd party cookie for global consent. In version 2 of the TCF Specifications, the storage mechanism used for service-specific TC Strings is up to a CMP, including any non-cookie storage mechanism.
```
Comment (a2intl) [inline TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md:338]: Perhaps mention that previous revisions of version 2 also used a 3rd party cookie for global consent.

File: TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md [hunk 4]
```diff
@@ -375,15 +348,6 @@ The following table summarises where data is stored:
     </tr>
   </thead>
   <tbody>
-    <tr>
```
Comment (a2intl) [inline TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md:378]: this table, now with one non-header row, seems to serve little purpose now.

File: TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md [hunk 5]
```diff
@@ -409,87 +373,19 @@ The IAB Europe Transparency & Consent Framework [Policies](https://iabeurope.eu/
...
+The _**[Core String](#the-core-string)**_ is always required and comes first and includes all the details required for communicating basic vendor transparency and consent. A TC String with all four segments is possible in certain conditions.
 
 ```
 COw4XqLOw4XqLAAAAAENAXCAAAAAAAAAAAAAAAAAAAAA.IFukWSQgAIQwgI0QEByFAAAAeIAACAIgSAAQAIAgEQACEABAAAgAQFAEAIAAAGBAAgAAAAQAIFAAMCQAAgAAQiRAEQAAAAANAAIAAggAIYQFAAARmggBC3ZCYzU2yIA.QFukWSQgAIQwgI0QEByFAAAAeIAACAIgSAAQAIAgEQACEABAAAgAQFAEAIAAAGBAAgAAAAQAIFAAMCQAAgAAQiRAEQAAAAANAAIAAggAIYQFAAARmggBC3ZCYzU2yIA.YAAAAAAAAAAAAAAAAAA
 ```
-A service-specific TC String must contain a Core TC String and may optionally contain a _**[Publisher TC](#publisher-purposes-transparency-and-consent)**_ segment, but must not contain the OOB-related segments because those segments are not allowed in service-specific contexts:
+A service-specific TC String must contain a Core TC String and may optionally contain a _**[Publisher TC](#publisher-purposes-transparency-and-consent)**_ segment :
```
Comment (a2intl) [inline TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md:388]: Perhaps remove "service-specific", as there isn't another kind now.

**Review by a2intl — COMMENTED** [review]
Suppose someone found an old, out-of-compliance, v2 consent string with an OOB or other deprecated segment. How would they know that they need to dig up old versions of the spec in order to decode it?

**Comment by anderagakura** [comment]
@alextcone the changes from #300 was not as small as they were supposed to. While I was doing the changes with Anne and Ninon (with Achim's update) in #301, the changes removed and broke my current changes within this #301. 

Therefore, I had to take account all the changes to do a new commit. Checking with other folks to see if we're ok. @asr-enid let us if this sounds ok for you as well

File: TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md [hunk 6]
```diff
@@ -1165,80 +1023,10 @@ CLcVDxRMWfGmWAVAHCENAXCkAKDAADnAABRgA5mdfCKZuYJez-NQm0TBMYA4oCAAGQYIAAAAAAEAIAEg
...
-
-```
-CGL23UdMFJzvuA9ACCENAXCEAC0AAGrAAA5YA5ht7-_d_7_vd-f-nrf4_4A4hM4JCKoK4YhmAqABgAEgAA.IFut_a83_Ma_t-_SvB3v4-IAeIAACAIgSAAQAIAgEQACEABAAAgAQFAEAIAAAGBAAgAAAAQAIFAAMCQAAgAAQiRAEQAAAAANAAIAAggAIYQFAAARmggBC3ZCYzU2yIA.QFulWfTw4obx_Z2zUj6XkNIAeIAACAIgSAAQAIAgEQACEABAAAgAQFAEAIAAAGBAAgAAAAQAIFAAMCQAAgAAQiRAEQAAAAANAAIAAggAIYQFAAARmggBC3ZCYzU2yIA
-```
-
 #### Disclosed Vendors
 
-The _**DisclosedVendors**_ is a TC String segment that signals which vendors have been disclosed to a given user by a CMP. This segment is required when saving a global-context TC String.  When a CMP updates a globally-scoped TC String, the CMP <span style="text-decoration:underline;">MUST</span> retain the existing values and only add new disclosed Vendor IDs that had not been added by other CMPs in prior interactions with this user.
+The _**DisclosedVendors**_ is an optional TC String segment that signals which vendors have been disclosed to a given user by a CMP. It may be used by a CMP while [storing](#how-should-a-transparency--consent-string-be-stored) TC Strings to record which vendors have been disclosed to the user, but must not be included in the TC String when returned by the CMP API.
```
Comment (asr-enid) [inline TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md:1028]: I'd suggest the following such that we do not use 'signal' to avoid any sort of potential misinterpretation, until we proceed on the discussion if and when this segment might be returned by the CMP API in the future

```suggestion
The _**DisclosedVendors**_ is an optional TC String segment that records which vendors have been disclosed to a given user by a CMP. It may be used by a CMP while [storing](#how-should-a-transparency--consent-string-be-stored) TC Strings, but must not be included in the TC String when returned by the CMP API.
```

File: TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md [hunk 7]
```diff
@@ -176,7 +177,7 @@ A TC String that contains positive consent signals must not be created before cl
 
 ### What is the scope for a TC String?
 
-CMPs must be set up to operate in a **service-specific**. A  TC String in this context is only used by the site(s) or app(s) on which it is running. One is created for every user on a given site/app or group of sites/apps. They may contain [Publisher restrictions](#what-are-publisher-restrictions), a _**[Publisher TC](#publisher-purposes-transparency-and-consent)**_ segment and an _**[AllowedVendors](#allowed-vendors-oob)**_ segment.
+CMPs must be set up to operate in a **service-specific** or group-specific configuration. A TC String in this context is applicable only on a service or group of services, for example the site(s) or app(s) on which it is running. One is created for every user on a given site/app or group of sites/apps. They may contain [Publisher restrictions](#what-are-publisher-restrictions) and a _**[Publisher TC](#publisher-purposes-transparency-and-consent)**_ segment when returned by the CMP API.
```
Comment (asr-enid) [inline TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md:180]: Formatting suggestion, I think we are using bold/italic for all the cross-references. 

```suggestion
CMPs must be set up to operate in a **service-specific** or group-specific configuration. A TC String in this context is applicable only on a service or group of services, for example the site(s) or app(s) on which it is running. One is created for every user on a given site/app or group of sites/apps. They may contain _**[Publisher restrictions](#what-are-publisher-restrictions)**_ and a _**[Publisher TC](#publisher-purposes-transparency-and-consent)**_ segment when returned by the CMP API.
```

File: TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md [hunk 8]
```diff
@@ -375,17 +347,18 @@ The IAB Europe Transparency & Consent Framework [Policies](https://iabeurope.eu/
 
 ### TC String Format
 
-There are 2 distincts TC String segments that are joined together on a “dot” character.  They are:
+There are 3 distincts TC String segments that are joined together on a “dot” character.  They are:
 
-*   The core vendor transparency and consent details and
+*   The core vendor transparency and consent details
+*   Disclosed vendors for solving ambiguous negative vendor signal and
```
Comment (asr-enid) [inline TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md:353]: Should we just leave it with Disclosed vendors (and the explanation in the segment description) for now until we figure out the signalling aspect of this? Given there is no expiation on how to use it for disambiguation that might be misleading for now. 

```suggestion
*   Disclosed vendors
```

**Review by asr-enid — COMMENTED** [review]
Thanks @anderagakura just a few small remarks

**Comment by asr-enid** [comment]
Thanks @anderagakura  looks good to me obviously w.r.t. merging #300

**Comment by anderagakura** [comment]
@asr-enid perfect

@dmdabbs @alextcone Following this PR, #303 #304 are in charge to update other pages from the current folder

**Comment by alextcone** [comment]
> @dmdabbs @alextcone Following this PR, #303 #304 are in charge to update other pages from the current folder

To be clear, there are 3 PRs for this one update? I think future readers of spec changes might find this a little cumbersome to follow. Was this broken up intentionally or more so due to the need to incorporate @asr-enid's previous changes to DisclosedVendors?

File: TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md [hunk 9]
```diff
@@ -180,16 +175,9 @@ A Transparency & Consent String may only be created by an IAB Europe TCF registe
...
+CMPs must be set up to operate in a **service-specific** or group-specific configuration. A TC String in this context is applicable only on a service or group of services, for example the site(s) or app(s) on which it is running. One is created for every user on a given site/app or group of sites/apps. They may contain _**[Publisher restrictions](#what-are-publisher-restrictions)**_ and a _**[Publisher TC](#publisher-purposes-transparency-and-consent)**_ segment when returned by the CMP API.
```
Comment (alextcone) [inline TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md:180]: I think it would be helpful to have a short statement here that acknowledges there once was a global scope, what its function was, and that it was deprecated September 1, 2021.

File: TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md [hunk 10]
```diff
@@ -343,15 +318,11 @@ The service making the call must replace the macros with appropriate values desc
...
+ If a publisher is operating a CMP within a jurisdiction that does not require consent to store and/or access information on a device and, therefore, does not ask for consent on behalf of a vendor for a purpose 1, the CMP will write the corresponding bit in the _**PurposesConsent**_ field to `0`. Even though it is valid within that jurisdiction to not request consent for Purpose 1, a vendor would interpret that `0` as a “no consent” signal and have no way of knowing that consent was not required in the jurisdiction in which the publisher operates.  This lack of transparency would, ultimately, cause losses in ad revenue for that publisher.
```
Comment (alextcone) [inline TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md:325]: This change seems unrelated to the Global Scope change, no?

File: TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md (hunk 4 above)
Comment (alextcone) [reply TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md:379]: Agree. It would make sense to pull the information out of tabular form.

File: TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md [hunk 11]
```diff
@@ -361,136 +332,35 @@ The following details provide information on creating, storing, and managing a T
...
-      <td></td>
-    </tr>
-  </tbody>
-</table>
-
-
-#### Global Cookie Storage Update (December 2019)
-- All requests that read from or write to the global cookie in the consensu.org domain must be secured by HTTPS _**(by July 31st 2021, the Managing Organisation (MO) will no longer delegate a subdomain of the consensu.org domain to each newly registered CMP and CMPs must no longer write TC Strings in the 'euconsent-v2' cookie associated with the consensu.org domain. From Sept 1st 2021, CMPs must no longer read and process TC Strings in the ‘euconsent-v2’ cookie associated with the consensu.org domain)**_
-- Additionally, browser cookie policies may require the support of certain attributes (e.g. sameSite, Secure)
+Post 30 September 2020, [v1.x strings were considered invalid](https://iabeurope.eu/all-news/the-iab-europe-transparency-consent-framework-tcf-steering-group-votes-to-extend-technical-support-for-tcf-v1-1/). If a CMP encounters a situation where both a v1.x string and a v2.0 string are erroneously present simultaneously, the CMP should remove the v1.x string to ensure that there is only one source of truth for consumers of the string.
```
Comment (alextcone) [inline TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md:345]: Is this new language? It isn't related to the Global Scope change so it seems messy to include it in this update if so.

File: TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md [hunk 12]
```diff
@@ -361,136 +332,35 @@ The following details provide information on creating, storing, and managing a T
...
+Note: TCF version 2.0 introduces [“Publisher Restrictions”](#what-are-publisher-restrictions), which, if exhausted by a publisher, could result in TC strings that are larger than the size limit for cookies. While this possibility is remote, it should be guarded against – a CMP should work with a publisher to help them accomplish their goals. CMPs may need to take this into consideration when deciding on the storage mechanism for those TC Strings.
 
 
 ### TC String Format
 
-There are 4 distinct TC String segments that are joined together on a “dot” character.  They are:
+There are 3 distincts TC String segments that are joined together on a “dot” character.  They are:
 
 *   The core vendor transparency and consent details
 *   Disclosed vendors
```
Comment (alextcone) [inline TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md:355]: This isn't staying in for signaling purposes, right, @asr-enid?

File: TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md (hunk 10 above)
Comment (anderagakura) [reply TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md:325]: From Ninon "_Yes indeed this is unrelated to the deprecation of global scope & OOB but it's necessary to make the change as the previous version mentioning legitimate interest does not make sense from a legal perspective. The change remains quite minimal and does not change the explanation as to how this field should be populated._"

File: TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md (hunk 11 above)
Comment (anderagakura) [reply TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md:345]: From Ninon "_Also unrelated to the deprecation of global scope & OOB - although the change is minimal, we just removed "before" by "post" and "will be" by "were" as we are now post September 2020._"

File: TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md (hunk 9 above)
Comment (AnneGoodman-IABEurope) [reply TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md:180]: Agree and propose:
What happened to Global scope and Out-of-Band ?

The TCF Policy previously allowed legal bases in the Framework to be established with global scope, which means a legal basis is not only applicable on the service or group of services (service-specific and group-specific scopes) where it is obtained and managed, but all services implementing global scope preferences. In the context of global scope, TC Strings were stored in a 3rd party cookie associated with the “consensu.org” domain that enabled Consent Management Providers (CMPs) to read the “TC strings from their subdomains [cmp-name].mgr.consensu.org across different services. Deprecation of global-scope support was announced on June 22nd 2021. Alongside the deprecation of global scope, support for Out-of-Band (OOB) - which refers to instances where a legal basis has not been established using the TCF in a global-scope context - was also deprecated. Since Sept 1st 2021, TC strings established with global-scope are considered invalid."

File: TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md (hunk 12 above)
Comment (asr-enid) [reply TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md:355]: Nope - its just kept here in the listing as this generally lists all segments. The explanation on disclosed vendors makes it explicit that its only usable for storage purposes (optional) - if you'd like on could add

```suggestion
*   Disclosed vendors (optional - not included when returned by CMP API)
```
but honestly that makes it more complicated that needed I guess

File: TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md (hunk 9 above)
Comment (alextcone) [reply TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md:181]: Added

**Comment by alextcone** [comment]
> > @dmdabbs @alextcone Following this PR, #303 #304 are in charge to update other pages from the current folder
> 
> To be clear, there are 3 PRs for this one update? I think future readers of spec changes might find this a little cumbersome to follow. Was this broken up intentionally or more so due to the need to incorporate @asr-enid's previous changes to DisclosedVendors?

Going to combine these into one branch and then merge after resolving comments.

**Comment by alextcone** [comment]
Close in favor of #305 which combines latest #301, #303, and #304
